### DISCUSSION TITLE
What is that unregistered thing?

### DISCUSSION THREAD

**Discussion by stdedos** [pfcao/abookmark#18] [Q&A]
![image](https://github.com/user-attachments/assets/47731ad2-6c37-4f06-bd78-fb9877eb2a03)

I've also seen https://www.abookmark.org/pricing - but ofc those P-things do not mean much 😓

**Comment by pfcao** [comment]
This is just to make it clear that this software is not free, but can be tried for free. Several paid plans will be available in the future.

**Reply by stdedos** [reply]
Aaah, right ... You don't have this open-sourced 😢 

I was misled by the few (one?) js file you have on the repo that you had this as open source 😕 
Too bad, as eventually I'd like to make this working a bit better _for me_ - as it is (naturally) highly opinionated
